RepoCleanup/Functions/CreateOrgWithTeams.cs

```diff
@@ -0,0 +1,152 @@
+
+            if (response.Success)
+            {
+                logBuilder.AppendLine($"{DateTime.Now} - Information - Org {org.Username} was successfully added to Gitea.");
+                createdOrg = true;
+            }
+            else
+            {
+                logBuilder.AppendLine($"{DateTime.Now} - Error - Org {org.Username} was not added to Gitea.");
+                logBuilder.AppendLine($"{DateTime.Now} - Error -  {response.StatusCode}:{JsonSerializer.Serialize(response.ResponseMessage, Globals.SerializerOptions)}.");
+            }
+
+            using (StreamWriter file = new StreamWriter(logName, true))
+            {
+                file.WriteLine(logBuilder.ToString());
+            }
+
+            logBuilder.Clear();
+
+            if (createdOrg)
+            {
+                Console.WriteLine($"Org {org.Username} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine($"Create org for {org.Username} failed. See log {logName}.");
+            }
+
+            bool createdTeams = true;
+            List<CreateTeamOption> teams = new List<CreateTeamOption>();
+            teams.Add(CreateTeamInfo("Deploy-Prod", "Members can deploy to production", false, Permission.read));
+            teams.Add(CreateTeamInfo("Deploy-TT02", "Members can deploy to TT02", false, Permission.read));
+            teams.Add(CreateTeamInfo("Devs", "All application developers", true, Permission.write));
+
+            Console.WriteLine("Creating teams for org. Please wait.");
+            foreach (CreateTeamOption team in teams)
+            {
+                response = await GiteaService.CreateTeam(org.Username, team);
+
+                if (response.Success)
+                {
+                    logBuilder.AppendLine($"{DateTime.Now} - Information - Team {team.Name} was successfully added to org: {org.Username}.");
+                }
+                else
+                {
+                    createdTeams = false;
+                    logBuilder.AppendLine($"{DateTime.Now} - Error - Team {team.Name} was not added to org {org.Username}.");
+                    logBuilder.AppendLine($"{DateTime.Now} - Error -  {response.StatusCode}:{JsonSerializer.Serialize(response.ResponseMessage, Globals.SerializerOptions)}.");
+                }
+            }
+
+            if (createdTeams)
+            {
+                Console.WriteLine($"All teams for {org.Username} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine($"Create teams for {org.Username} failed. See log {logName}.");
+            }
+        }
+
+        private static Organisation CollectOrgInfo()
+        {
+            bool isValid = false;
+            string username = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet username (shortname) for org: ");
+                username = Console.ReadLine();
```

Comment:
Does gitea ignore the case or should we convert it all to lower case just in case?

Reply:
Lower case it is.

Reply:
@tba76 Lowercase is not enough :)

https://github.com/Altinn/altinn-studio/blob/42a4ba3e1d16c8c13b97acf63925cf81f601f281/src/studio/src/designer/frontend/dashboard/features/createService/createNewService.tsx#L130

![image](https://user-images.githubusercontent.com/6088624/107637555-94961c00-6c6e-11eb-86a2-43b575624011.png)

Reply:
Regex changed.